examples/camera-app/camera-common/include/camera-device-interface.h

```diff
@@ -113,16 +113,17 @@ class CameraDeviceInterface
     virtual ~CameraDeviceInterface() = default;
 
     // Getter for Chime Delegate
-    virtual chip::app::Clusters::ChimeDelegate & GetChimeDelegate() = 0;
+    virtual chip::app::Clusters::ChimeController & GetChimeDelegate() = 0;
 
     // Getter for WebRTCProvider Delegate
-    virtual chip::app::Clusters::WebRTCTransportProvider::Delegate & GetWebRTCProviderDelegate() = 0;
+    virtual chip::app::Clusters::WebRTCTransportProvider::WebRTCProviderController & GetWebRTCProviderDelegate() = 0;
 
     // Getter for CameraAVStreamManagement Delegate
-    virtual chip::app::Clusters::CameraAvStreamManagement::CameraAVStreamMgmtDelegate & GetCameraAVStreamMgmtDelegate() = 0;
+    virtual chip::app::Clusters::CameraAvStreamManagement::CameraAVStreamController & GetCameraAVStreamMgmtDelegate() = 0;
```

Comment:
These Getters should also be renamed accordingly? GetCameraAVStreamManager()?

Reply:
We can not GetCameraAVStreamManager since it is deifned in linux/include, and linux could include the headers from camera-common but not the other way around.
What we could do is:
 virtual chip::app::Clusters::CameraAvStreamManagement::CameraAVStreamMgmtDelegate & GetCameraAVStreamMgmtDelegate() = 0;
 virtual chip::app::Clusters::CameraAvStreamManagement::CameraAVStreamController & GetCameraAVStreamMgmtController() = 0;